src/gtest/test_checkblock.cpp

```diff
@@ -93,7 +93,8 @@ class ContextualCheckBlockTest : public ::testing::Test {
 
     void TearDown() override {
         // Revert to test default. No-op on mainnet params.
-        RegtestDeactivateSapling();
+        RegtestDeactivateBlossom();
+        UpdateNetworkUpgradeParameters(Consensus::UPGRADE_BLOSSOM, Consensus::NetworkUpgrade::NO_ACTIVATION_HEIGHT);
```

Comment:
```suggestion
```
RegtestDeactivateBlossom already does this; why does it need to be done again?

Reply:
Good catch, i messed up my commits/branching work and also wasn't sure if `RegtestDeactivateBlossom` was OK to use given [it also does](https://github.com/zcash/zcash/blob/e568a190f3d56440820dab6b7c434b8d9800adee/src/utiltest.cpp#L233-L239):

```
UpdateRegtestPow(0, 0, uint256S("0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f"), true);
SelectParams(CBaseChainParams::MAIN);
```

The RegtestPow thing seems pretty inoffensive since that's the default value for Regtest but i'm unsure what the `SelectParams(CBaseChainParams::MAIN);` line is doing -- there's no [equivalent line](https://github.com/zcash/zcash/blob/e568a190f3d56440820dab6b7c434b8d9800adee/src/utiltest.cpp#L217-L220) in `RegtestDeactivateSapling`

Reply:
i think what had worked was to keep the `RegtestDeactivateSapling` and only add the `UpdateNetworkUpgradeParameters(Consensus::UPGRADE_BLOSSOM, Consensus::NetworkUpgrade::NO_ACTIVATION_HEIGHT);` afterwards. i'm going to re-test that to be sure.